openfisca_core/scripts/migrations/v24_to_25.py

```diff
@@ -52,7 +52,7 @@ def migrate(self, path):
         print('Migrating {}.'.format(path))
 
         with open(path) as yaml_file:
-            tests = yaml.load(yaml_file)
+            tests = yaml.load(yaml_file, Loader=yaml.FullLoader)
```

Comment:
I don't think we really need to use the unsafe loader.

We only use YAML files as human-readable JSONs, and none of the more advanced unsafe features.

We could just replace the `yaml.load` by `yaml.safe_load`, and avoid having to specify a loader, which is a bit heavy. I tried on [this branch](https://github.com/openfisca/openfisca-core/compare/explicit-yaml-loader...explicit-yaml-loader-safe) and the [tests pass](https://circleci.com/gh/openfisca/openfisca-core/4692).